open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/IntegrationViewServiceConfig.java

```diff
@@ -49,6 +50,17 @@ public IntegrationViewServiceConfig(IntegrationViewServiceConfig template)
         }
     }
 
+    /**
+     * Call the superclass to set up the default values for a view service using a view service description.
```

Comment:
registration not description

Reply:
I used the same comment (almost) as in the ViewServiceConfig. Happy to change it if important but I thought it made sense anyway